Dockerfile

```diff
@@ -28,6 +28,9 @@ RUN chown -R invenio:invenio ${WORKING_DIR}
 
 USER 1000
 
+# Get last GIT commit hash
+ENV INVENIO_RERO_ILS_APP_GIT_HASH $(git rev-parse @)
```

Comment:
This will not work as the .git does not exists on the Docker image. It would be better to use ARG: Set build-time variables (--build-arg) section in https://docs.docker.com/engine/reference/commandline/build. This is also important to have the choice to set this variable as we do not want to do it for production!

You can try by yourself by building the docker image and run a command inside:
```bash
docker build --rm -t rero/rero-ils:test .
docker run --rm -ti rero/rero-ils:test git rev-parse @
or
docker run --rm -ti rero/rero-ils:test /bin/bash
and
echo $INVENIO_RERO_ILS_APP_GIT_HASH
```
The last command return `$(git rev-parse @)` which is not correct.